core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.css

```diff
@@ -129,7 +129,7 @@
 /*rtl:raw:
 .navbar-header {
   display: flex;
-  margin-left: -30px;
+  margin-left: -10px;
 }
 */
```

Comment:
Remove commented out code.
Also, there should be no cases where we have exceptions for rtl css. The rtl version of the normal css should work as is, unless it in unavoidable. Why is this in a pragma?

Reply:
@aks681 There was some issue while I was viewing the changes in the frontend and due to few other issues, the view was breaking in rtl view, that's why had added them, but I have updated the whole PR and removed the exceptions for rtl that were not required. Thanks!